pkgs/kde/third-party/koi/default.nix

```diff
@@ -19,8 +20,18 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-dhpuKIY/Xi62hzJlnVCIOF0k6uoQ3zH129fLq/r+Kmg";
   };
 
+  patches = [
+    ./patches/0001-plasmastyle.cpp-Use-relative-path.patch
+    ./patches/0002-icons.cpp-Use-configurable-libexec-path.patch
+  ];
+  postPatch = ''
+    sed -i "s|@plasma-changeicons@|${plasma-workspace}/libexec/plasma-changeicons|g" src/icons.cpp
+  '';
   # See https://github.com/baduhai/Koi/blob/master/development/Nix%20OS/dev.nix
-  sourceRoot = "source/src";
+  sourceRoot = "source";
+  preConfigure = ''
+    cd src
+  '';
```

Comment:
Lets rather strip src from the patches

Reply:
Doing that makes it a lot more annoying to create git patches.